.github/workflows/stackhpc-ci-cleanup.yml

```diff
@@ -75,3 +75,23 @@ jobs:
           OS_CLOUD: openstack
           OS_APPLICATION_CREDENTIAL_ID: ${{ secrets.OS_APPLICATION_CREDENTIAL_ID }}
           OS_APPLICATION_CREDENTIAL_SECRET: ${{ secrets.OS_APPLICATION_CREDENTIAL_SECRET }}
+
+      - name: Clean up periodic multinode instances over 12 hours old
+        run: |
+          result=0
+          changes_before=$(date -Imin -d -12hours)
+          for status in ACTIVE BUILD ERROR SHUTOFF; do
+              for instance in $(openstack server list --tags skc-multinode-periodic --os-compute-api-version 2.66 --format value --column ID --changes-before $changes_before --status $status); do
```

Comment:
TODO: add this tag to multinode instances.

Reply:
Do you want to add this before merging this one?

Reply:
But probably better to keep this open as a reminder to do it